python_ta/debug/snapshot.py

```diff
@@ -3,10 +3,12 @@
  multiple stack frames. Useful for dynamic debugging.
 """
 import inspect
+import json
 from types import FrameType
+from typing import Dict, List
```

Comment:
You don't need these type annotations. Instead, add the import `from __future__ import annotations` at the top of this file, and you can then use the lowercase versions of `list` and `dict`

Reply:
Hello @david-yz-liu, Thank you for providing feedback. I implemented these changes in response to certain tests failing on Python version 3.8. Do you think it would be advisable to maintain these modifications? 

![image](https://github.com/pyta-uoft/pyta/assets/121046082/e1e13343-36b3-47a5-a0d7-7b404660bde1)

Reply:
@umututku03 do what I've said to do and the test should not be an issue